### ISSUE TITLE
Personal Assistant Project will only work on windows

### ISSUE THREAD

**Issue by genbod** [microsoft/botframework-solutions#3253]
#### What project is affected
Virtual Assistant
#### What language is this in?
node.js
#### What happens?
when i18next loads it looks for language files like this: en-US.json etc.  However, the project has them named like this: en-us.json.  That works ok in windows because window file access is case-insensitive.  However, it breaks in linux in a very frustrating way because none of the languages load etc.

#### What are the steps to reproduce this issue?
Run node.js app in linux container
#### What were you expecting to happen?
Default app works.

**Comment by Batta32** [comment]
Thanks @genbod for reporting this issue! We will be reviewing it and we will back to you later 😊.

**Comment by VictorGrycuk** [comment]
Hi @genbod, sorry for the delay. We created the PR [#3288](https://github.com/microsoft/botframework-solutions/pull/3288) working with case-sensitive language codes. Can you confirm that these changes works for you?

1. Clone the branch `feature/southworks/case-sensitiveness/fix-language-code`
1. Go to the Virtual Assistant Sample's folder
1. Either use already existing resources or deploy a new Virtual Assistant
1. Install the dependencies executing `npm install`
1. Start the Virtual Assistant executing `npm start`
1. Check that the issue is fixed

We added the `lowerCaseLng: true` when the `i18next` configuration is loaded in the Virtual Assistant and the Skill Sample and Template as well.

```typescript
i18next.use(i18nextNodeFsBackend)
    .init({
        lowerCaseLng: true,
        fallbackLng: 'en-us',
        preload: ['de-de', 'en-us', 'es-es', 'fr-fr', 'it-it', 'zh-cn'],
        backend: {
            loadPath: path.join(__dirname, 'locales', '{{lng}}.json')
        }
    })
    .then(async (): Promise<void> => {
        await Locales.addResourcesFromPath(i18next, 'common');
    });
```

Last but not least, the issue didn't happen using the next daily builds of the Virtual Assistant, Skill and BotBuilder-Solutions library:
- [generator-botbuilder-assistant@1.0.9-preview171](https://botbuilder.myget.org/feed/aitemplates/package/npm/generator-botbuilder-assistant/1.0.9-preview171)
- [botbuilder-solutions 4.5.0-preview223](https://botbuilder.myget.org/feed/aitemplates/package/npm/botbuilder-solutions/4.5.0-preview223)

You can test them with the following steps:
1. Run `npm uninstall -g generator-botbuilder-assistant` to uninstall the generator if you already have it installed
1. Install the generator by running `npm install -g generator-botbuilder-assistant@1.0.9-preview171 --registry https://botbuilder.myget.org/F/aitemplates/npm`
1. Run `yo botbuilder-assistant` and follow the steps to generate a new assistant
1. Once the Sample Assistant is generated, you have to make it use the daily of `botbuilder-solutions`. You have two options (take into account to delete the `npm_modules` and the `package-lock.json`):
   - Option 1: Add a `.npmrc` file pointing to the registry `https://botbuilder.myget.org/F/aitemplates/npm`, and updating the `package.json` to point the `botbuilder-solutions` version to `4.5.0-preview223`
   - Option 2: Download the tgz of `botbuilder-solutions` referencing it directly in the `package.json` with an absolute path
1. Install the dependencies
1. Either use already existing resources or deploy a new Virtual Assistant
1. Run the Sample Assistant

Please let us know if this fixes the issue of the locales, or if there are additional case-sensitive issues.

Thanks.

**Comment by peterinnesmsft** [comment]
@genbod can you take a look at the draft PR and let us know if the proposed solution will address your raised issue?

**Comment by genbod** [comment]
Yes.  This looks like it will fix the issue.  I left a comment in the PR.  Is there any work around testing in linux and windows going forward?

**Comment by peterinnesmsft** [comment]
I took a look at the PR and signed off. @Batta32 let us know when you can get the change merged etc.

@genbod I will follow up with the team offline to see what we can do to incorporate wider testing against Linux as well as Windows going forward.

**Comment by Batta32** [comment]
@peterinnesmsft - PR #3288 is ready to be merged to the branch `master` in order to solve this issue.

Also, the same changes were implemented to `next` for the next release PR #3305.
